files/whitelist

```diff
@@ -20,7 +20,7 @@ begin
   File.open(path, 'w') {|f| f.write whitelisted_ips.to_yaml }
 
   puts "Stopping Rails server. Please restart it."
-  `kill -9 $(lsof -i tcp:8080 -t) >/dev/null 2>/dev/null`
+  `kill -9 $(lsof -i tcp:3000 -t) >/dev/null 2>/dev/null`
```

Comment:
@jelaniwoods This might be for a separate patch, but this reminds me that we should update `bin/server` to also kill 3000; and also to `rails server -b 0.0.0.0`

Reply:
@raghubetina Oh I didn't realize this was also in `bin/server`. That change seems closely related to this pr. I'll add it. Where did you want the `rails server -b 0.0.0.0`?

Reply:
Oh wait I see now.

Reply:
@jelaniwoods Also in `bin/server`, replacing Cloud9's `$IP -p $PORT`